app/models/receipt.rb

```diff
@@ -4,4 +4,6 @@ class Receipt < ApplicationRecord
 
   enum status: {wait: 0, approved: 1, complete: 2,
                 cancelled_by_admin: 3, cancelled_by_you: 4}
+  scope :status_priority, ->{order status: :asc}
```

Comment:
```suggestion
  scope :status_priority, ->{order :status}
```
asc là default nên mạnh dạn lượt bỏ